ibis-server/tests/routers/v2/connector/test_oracle.py

```diff
@@ -0,0 +1,407 @@
+            "manifestStr": manifest_str,
+            "parameters": {"modelName": "Orders", "columnName": "orderkey"},
+        },
+    )
+    assert response.status_code == 204
+
+
+async def test_validate_rule_column_is_valid_with_invalid_parameters(
+    client, manifest_str, oracle: OracleDbContainer
+):
+    connection_info = _to_connection_info(oracle)
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={"connectionInfo": connection_info, "manifestStr": manifest_str},
+    )
+    assert response.status_code == 422
+    result = response.json()
+    assert result["detail"][0] is not None
+    assert result["detail"][0]["type"] == "missing"
+    assert result["detail"][0]["loc"] == ["body", "parameters"]
+    assert result["detail"][0]["msg"] == "Field required"
+
+
+async def test_validate_rule_column_is_valid_without_parameters(
+    client, manifest_str, oracle: OracleDbContainer
+):
+    connection_info = _to_connection_info(oracle)
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={"connectionInfo": connection_info, "manifestStr": manifest_str},
+    )
+    assert response.status_code == 422
+    result = response.json()
+    assert result["detail"][0] is not None
+    assert result["detail"][0]["type"] == "missing"
+    assert result["detail"][0]["loc"] == ["body", "parameters"]
+    assert result["detail"][0]["msg"] == "Field required"
+
+
+async def test_validate_rule_column_is_valid_without_one_parameter(
+    client, manifest_str, oracle: OracleDbContainer
+):
+    connection_info = _to_connection_info(oracle)
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={
+            "connectionInfo": connection_info,
+            "manifestStr": manifest_str,
+            "parameters": {"modelName": "Orders"},
+        },
+    )
+    assert response.status_code == 422
+    assert response.text == "Missing required parameter: `columnName`"
+
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={
+            "connectionInfo": connection_info,
+            "manifestStr": manifest_str,
+            "parameters": {"columnName": "orderkey"},
+        },
+    )
+    assert response.status_code == 422
+    assert response.text == "Missing required parameter: `modelName`"
+
+
+async def test_metadata_list_tables(client, oracle: OracleDbContainer):
+    connection_info = _to_connection_info(oracle)
+    response = await client.post(
+        url=f"{base_url}/metadata/tables",
+        json={"connectionInfo": connection_info},
+    )
+    assert response.status_code == 200
+
+    result = response.json()
+    # Use case-insensitive filtering since Oracle returns "SYSTEM.ORDERS"
+    result = next(filter(lambda x: x["name"].lower() == "system.orders", result))
+
+    assert result["name"].lower() == "system.orders"
```

Comment:
```suggestion
    result = next(filter(lambda x: x["name"] == "SYSTEM.ORDERS", result))
    assert result is not None
```
We can just match the upper case name and assert if it's None.